lang/java/reef-io/src/main/java/org/apache/reef/io/network/group/impl/task/OperatorTopologyStructImpl.java

```diff
@@ -234,6 +277,24 @@ private void sendToNode(final byte[] data, final ReefNetworkGroupCommProtos.Grou
         node, remove}));
     return retVal;
   }
+  
+  private byte[][] receiveArrayFromNode(final NodeStruct node, final boolean remove) {
```

Comment:
I agree with you that the *Array* methods are twins of the original non-*Array* methods, with only the parameter `byte[]` changed to `byte[][]`. I'll rewrite the code to remove repetition and make another push.